### PR TITLE
Handling display of a Work when in Reading Room

### PR DESCRIPTION
**Pull request by adamjarling**
## What does this do?
Handles display of a Work when in the Reading Room.

#### Authenticated view
image

#### Reading Room view
image

### How to Test
- Run the local dev environment
- In `pages/_app.tsx`, on line 49, set the value of `isReadingRoom` to `true` to mock what the Reading Room auth request response would look like.
```jsx
...
setUser({
  email,
  isLoggedIn,
  isReadingRoom: true,
  name,
  sub,
});
```

### CODE REVIEW

File: components/Work/ViewerWrapper.styled.ts [hunk 1]
```diff
@@ -7,4 +7,16 @@ const ViewerWrapperStyled = styled("section", {
...
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+
+  "& svg": {
+    height: "$gr4",
+    fill: "$black80",
+    marginRight: "$gr1",
+  },
+});
```
Comment (mathewjordan) [inline components/Work/ViewerWrapper.styled.ts:20]: This looks good. 👨‍🍳

File: components/Work/ViewerWrapper.test.tsx [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+  user: {
+    email: "joan.doe@northwestern.edu",
+    isLoggedIn: true,
+    isReadingRoom: false,
+    name: "Joan Doe",
+    sub: "jdoe2399",
+  },
+};
+const readingRoomMessage =
+  /You have access to Work because you are in the reading room/i;
```
Comment (mathewjordan) [inline components/Work/ViewerWrapper.test.tsx:15]: Is this regex?

**Review by mathewjordan — APPROVED** [review]
Good from my eyes. That mocked user is something I want to wrap my mind around bit more.

File: components/Work/ViewerWrapper.test.tsx (hunk 2 above)
Comment (adamjarling) [reply components/Work/ViewerWrapper.test.tsx:15]: Yea, case insensitive to tests don't break if capitalization changes or something.  Not sure this text is 100% locked down.  @davidschober ?

**Pull request merged**